Add info for how to build from source

I can't build this because it can't find nuget packages, but I don't know how to add those

```
error NU1101: Unable to find package Resonite.Elements.Core. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages, nuget.org

error NU1101: Unable to find package MonkeyLoader. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages, nuget.org

error NU1101: Unable to find package MonkeyLoader.GamePacks.Resonite. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages, nuget.org

error NU1101: Unable to find package Resonite.FrooxEngine. No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages, nuget.org
```

--- a/README.md
+++ b/README.md
@@ -36,7 +36,9 @@ The short summary of it is:
 
 1. [Setup a private nuget feed](https://github.com/MonkeyModdingTroop/ReferencePackageGenerator).
 2. [Generate the game's reference assemblies](https://github.com/MonkeyModdingTroop/ReferencePackageGenerator).
-3. Run `dotnet build`, or build with your IDE of preference.
+3. Add the nuget feeds (`nuget sources Add -Name ... -Source ...`, local and either <https://pkg.munally.com/MonkeyModdingTroop/index.json> & <https://pkg.munally.com/ResoniteModdingGroup/index.json>)
+4. Run `dotnet build`, or build with your IDE of preference.
 
 The long version is that you'll probably want to set it up privately on GitHub nuget packages.
 Though this isn't legal advice and you should check that [Resonite's TOS](https://resonite.com/policies/TermsOfService.html) allows it.
+The feeds can also be directly used from GitHub, though that requires authentication using a PAT.

--- a/README.md
+++ b/README.md
@@ -9,11 +9,6 @@ This means that any RML mods from the `rml_mods` folder can be loaded directly t
 without adding RML as a library that [Resonite](https://resonite.com) should load.
 Any libraries from the `rml_libs` folder will be loaded as well.
 
-Because the RML mods are loaded as fully integrated MonkeyLoader mods,
-they are integrated into the MonkeyLoader Settings Category inside Resonite as well.  
-The settings files will be created under `./MonkeyLoader/Configs` however,
-so any customized settings will have to recreated or manually copied into the json file.
-
 ## Installation
 
 1. Download `MonkeyLoader-v...+Resonite-v....zip` [from the latest Resonite gamepack release](https://github.com/ResoniteModdingGroup/MonkeyLoader.GamePacks.Resonite/releases/latest)
@@ -23,4 +18,25 @@ so any customized settings will have to recreated or manually copied into the js
 
 ## Settings Integration
 
+Because the RML mods are loaded as fully integrated MonkeyLoader mods,
+they are integrated into the MonkeyLoader Settings Category inside Resonite as well.  
+The settings files will be created under `./MonkeyLoader/Configs` however,
+so any customized settings will have to recreated or manually copied into the json file.
+
+
 ![Screenshot of the Resonite Settings with an RML mod opened in the MonkeyLoader category.](https://raw.githubusercontent.com/ResoniteModdingGroup/MonkeyLoader.GamePacks.ResoniteModLoader/master/MonkeyLoaderSettings.png)
+
+## Contributing
+
+Issues can and should be opened here instead of the mods' issue trackers if they're designed for RML, and work with it, but not with this gamepack.
+The GitHub issues can also be used for feature requests.
+
+For code contributions, getting started is a bit involved due to [Resonite-Issues#456](https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/456).
+The short summary of it is:
+
+1. [Setup a private nuget feed](https://github.com/MonkeyModdingTroop/ReferencePackageGenerator).
+2. [Generate the game's reference assemblies](https://github.com/MonkeyModdingTroop/ReferencePackageGenerator).
+3. Run `dotnet build`, or build with your IDE of preference.
+
+The long version is that you'll probably want to set it up privately on GitHub nuget packages.
+Though this isn't legal advice and you should check that [Resonite's TOS](https://resonite.com/policies/TermsOfService.html) allows it.

